### ISSUE TITLE
3. Integrate WHO alerts

### ISSUE THREAD

**Issue by dereklieu** [IFRCGo/go-frontend#254]

**Comment by dereklieu** [comment]
Blocked by WHO alerts not being available. Emergencies are created from other entities. Closing.

**Comment by dereklieu** [comment]
This is out of scope for phase 2.

**Comment by ElsaRaunioIFRC** [comment]
There appears to be another, more active WHO RSS, that we could consider adding on GO:

https://www.who.int/feeds/entity/csr/don/en/rss.xml

**Comment by szabozoltan69** [comment]
I implemented a WHO-reader, the RSS data generate Emergency ( = event) entities.
What shall I use as alert level? The most data in the RSS has "news" type. Can I label the "news" with green and the other with orange? (Or is "green" not enough severe?)

**Comment by nanometrenat** [comment]
From @RFMoore 
WHO Health Emergencies dashboard (number of events and WHO classification. Updates every 15 minutes): https://extranet.who.int/publicemergency
...suggested replacements for WHO DONs

Further investigation required before taking any action, but pasting here for future reference.

**Comment by szabozoltan69** [comment]
Just a note: recently there is a WHO ingestor script (on backend side), which was done in Phase3, and was run even in production, but now the cron job is not running. (It's just depend on a decision, that "let's use it!")
It gets data from WHO RSS feed: https://github.com/IFRCGo/go-api/blob/master/api/management/commands/ingest_who.py#L21

**Comment by nanometrenat** [comment]
Thanks Zoltan - it would be great if that WHO RSS feed had clearer indicators of the scale of the emergency - as you have identified earlier in the thread, 'news' is too generic.

That dashboard I've linked above has good categorisation but the problem is the data isn't accessible to my knowledge.  We need to dig further at this end.  Cheers!

**Comment by nanometrenat** [comment]
@RFMoore the source you sent through doesn't publish the data (can only look at the dashboard).  Do you have another source that we can read programmatically?  Thanks!

**Comment by nanometrenat** [comment]
Just checked with Roxanne, the WHO ingestor system needs to be put on hold until Health are able to confirm final severity criteria.  This goes alongside work they are doing on the Emergency Response Framework and IFRC's classification of diseases.

Thanks for your work on this @szabozoltan69

**Comment by szabozoltan69** [comment]
Ok, the ingestor still not runs, just a test run was issued in staging environment.

**Comment by LukeCaley** [comment]
@RFMoore - would there now be a way to connect to live WHO data around COVID-19 cases which this could connect to?

**Comment by nanometrenat** [comment]
Hi @LukeCaley this ticket (and the scraper/ingest) was set up to Create Emergencies - in the same way to GDACS - so I think if we want to instead track COVID cases it would be a from-scratch build.

Re this ticket (creating emergencies), I suggest we close/freeze it unless/until there is a data source available that we can rely on programmatically.  Best!

**Comment by LukeCaley** [comment]
Thanks @nanometrenat - understood the original intention but wondered if there was a country-level feed that would create emergencies on the first reported case or similar.  Not a priority, especially as NS are reporting snapshot figures through field reports for now.  Agreed that this is not a priority as of now and potentially to be revisited as part of scoping for ingest of health-related Common Alerting Protocol [https://en.wikipedia.org/wiki/Common_Alerting_Protocol] feed data.
